README.md

```diff
@@ -6,6 +6,16 @@ Shared metadata for Exercism exercises.
 
 Please see the [contributing guide](./CONTRIBUTING.md).
 
+## Reviewing Guidelines
+
+Pull Requests to this repo require three approving reviews to be merged.
+
+- If you think that a PR should not be merged, either as is or in general, please leave a review **requesting changes**.
+- Please also use the **Request Changes** button if you are unsure about how a concern of yours could be resolved.
+  - This will block the PR from being merged until the change request has been addressed / overruled by an admin.
+  - It ensures that your thoughts will be heard and can be addressed rather than being overruled by a majority.
+- When a discussion goes stale or there are several opposing views and a compromise cannot be reached, ping [@ErikSchierboom](https://github.com/ErikSchierboom/) to resolve the discussion.
```

Comment:
Let's make it the maintainers-admin group to futureproof having other people in that alongside Erik, and also so things get to me if he's on holiday etc. I don't think there's a great link for that, so I've removed it, but I have no problem with some link being added back if appropriate.

```suggestion
- When a discussion goes stale or there are several opposing views and a compromise cannot be reached, ping @exercism/maintainers-admin to resolve the discussion.
```

Reply:
I've added a link to the team page